docs/docs/overview/quickstart/get-started-with-React.md

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 3
+sidebar_label: Get started with React
+---
+
+# React Quickstart
+
+Learn how to integrate Novu into your React.js app on the fly. Send notifications across channels (SMS, Email, Chat, Push) with the help of a rich, customizable, real-time In-App notification center
+
+## Requirements
+
+To follow the steps in this quickstart, you'll need:
+
+- A Novu account. [Sign up for free](http://web.novu.co) if you don’t have one yet.
+- A working React development environment.
+
+You can find the code for this project [here](https://github.com/sumitsaurabh927/react-fe-for-quickstart)
```

Comment:
We would need to move the project code under our organization repositories.

Reply:
Done here d7c2076